lib/grp.gi

```diff
@@ -2959,11 +2955,7 @@ local n,a;
   fi;
 
   a:=AutomorphismGroup(G);
-  if ForAny(GeneratorsOfGroup(a),i->Image(i,H)<>H) then
-    return false;
-  else
-    return true;
-  fi;
+  return not ForAny(GeneratorsOfGroup(a),i->Image(i,H)<>H);
```

Comment:
Perhaps nice with fewer negatives:
```suggestion
  return ForAll(GeneratorsOfGroup(a),i->Image(i,H)=H);
```

Reply:
Good idea @fingolfin, I was trying to make this easy to review, so didn't make additional changes like this initially